sdk/scheduler/src/main/java/org/apache/mesos/specification/util/RLimit.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.mesos.specification.util;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Representation of an individual rlimit, consisting of a name and optional soft/hard limits.
+ *
+ * A valid instance of this class has either both limits set or neither, with the further constraint that the soft limit
+ * must be less than or equal to the hard limit.
+ */
+public class RLimit {
+    public static final String UNKNOWN = "RLIMIT_UNKNOWN";
```

Comment:
A user hopefully shouldn't be touching the `RLimit` class directly, but if they do it'd be nice to have the constants available on instantiation. I also forgot to use them in `ExecutorUtils`, but making that change now.